### PR TITLE
fix: Incorrect db user name in docker compose.

### PR DESCRIPTION
**Pull request by hampuslavin**
## Changes
- Fixes an error where the db user name was wrong in the docker compose

### CODE REVIEW

File: docs/06-concepts/18-testing/01-get-started.md [hunk 1]
```diff
@@ -124,10 +124,14 @@ volumes:
...
-# This is the configuration file for your local test environment. All ports are set to zero in this file which makes the server find the next available port. This is needed to enable running tests concurrently.
+# This is the configuration file for your test environment.
+# All ports are set to zero in this file which makes the server find the next available port.
+# This is needed to enable running tests concurrently. To set up your server, you will
 # need to add the name of the database you are connecting to and the user name.
 # The password for the database is stored in the config/passwords.yaml.
-
+#
+# When running your server locally, the server ports are the same as the public
+# facing ports.
```
Comment (SandPod) [inline docs/06-concepts/18-testing/01-get-started.md:134]: Question: What does this addition mean? This isn't really clear to me and might confuse a reader of the documentation as well.

**Review by SandPod — COMMENTED** [review]
One question, otherwise it looks good!

File: docs/06-concepts/18-testing/01-get-started.md (hunk 1 above)
Comment (hampuslavin) [reply docs/06-concepts/18-testing/01-get-started.md:134]: This was copy paste from the other config files, but you're right, this doesn't make sense for the test file. Will remove!

Comment (hampuslavin) [reply docs/06-concepts/18-testing/01-get-started.md]: e2f78d2

**Review by SandPod — APPROVED** [review]
LGTM! 👍

**Pull request merged**